src/containers/CardContainer.tsx

```diff
@@ -0,0 +1,47 @@
+import React, {Component} from 'react';
+import Cardgrid from 'components/Cardgrid';
+
+export default class CardContainer extends Component<any, any> {
+
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      data: [],
+      loading: true,
+    }
+    this.loadMore = this.loadMore.bind(this);
+  }
+
+  public componentDidMount() {
+    this.loadMore();
```

Comment:
Could you explain? [docs](https://facebook.github.io/react/docs/react-component.html#componentdidmount)